app/services/rapid_ready_for_decision/views/hypertension.erb

```diff
@@ -0,0 +1,5 @@
+<%= render_partial 'hypertension/intro' %>
```

Comment:
I think it would be cool to also make some of these partials more generic (looks like there's a shared folder). I got confirmation from Shannon that if the formatting is updated for the new contention, it's okay to apply it to the previous contention as well. She mentioned maybe wanting some differences with the medications though.

Reply:
@leikkisa great. yeah, I was able to make intro into something generic (see new updates)... i could see BP even being pretty generic even though it's only used for this currently